api/docs.go

```diff
@@ -2568,6 +2634,55 @@ const docTemplate = `{
                 }
             }
         },
+        "api.DetectRpmsRequest": {
+            "type": "object",
+            "properties": {
+                "limit": {
+                    "description": "Maximum number of records to return for the search",
+                    "type": "integer"
+                },
+                "rpm_names": {
+                    "description": "List of rpm names to search",
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "urls": {
+                    "description": "URLs of repositories to search",
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "uuids": {
+                    "description": "List of RepositoryConfig UUIDs to search",
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                }
+            }
+        },
+        "api.DetectRpmsResponse": {
+            "type": "object",
+            "properties": {
+                "found": {
+                    "description": "Package list of the package group found",
```

Comment:
i'm not sure why this shows up as a package list from the SearchPackageGroupResponse..

Reply:
I didn't see this comment before. I'll try to figure out what's going on.

Reply:
looks like using a model / changing the api response to []string fixed this :)